packages/commonwealth/client/scripts/navigation/helpers.tsx

```diff
@@ -80,6 +80,27 @@ const withRouter = (Component) => {
   };
 };
 
+export const navigateToPathInNewTab = ({
```

Comment:
added a tentative feature, if u click from feed in root URL to a community, opens in new tab, could revert. But generally could be useful